qemu_v8.mk

```diff
@@ -60,12 +65,29 @@ ARM_TF_FLAGS ?= \
 	DEBUG=$(ARM_TF_DEBUG) \
 	LOG_LEVEL=$(ARM_TF_LOGLVL)
 
+ifeq ($(CFG_SECURE_BOOT),1)
+ARM_TF_FLAGS += \
+	MBEDTLS_DIR=$(ARM_TF_PATH)/../mbedtls \
```

Comment:
This change expects mbedtls source tree from the project root dir?
If so, would prefer `$(ROOT)/mbedtls`.

Reply:
Ok. Will change.